documentation/docs/migration-guide.md

```diff
@@ -9,6 +9,71 @@ import { DemoWrapper } from '@site/docs/components/DemoWrapper';
 
 This document is an overview of some important differences between Reveal 3.x and Reveal 4.x.
 
+## Entrypoint and import paths
+For Reveal 4 we changed into using a single entry point for everything: `@cognite/reveal`.
+So extensions and tools also should be imported from this entry point.
+We have however also added support for node's module `exports` such that if your tsconfig is setup with:
+```ts
+"moduleResolution": "Node16",
+```
+
+imports will still work the same way they did with Reveal 3.x. 
+Note that this may require a bump in typescript version.
+If this is not possible for you, then you need to change all imports to be `from @cognite/reveal` vs. f.ex. `from @cognite/reveal/tools`. 
+
+## Three.js is now a peer-dependency
+For Reveal 4 we have changed threejs from being exported from Reveal and rather require applications that consume Reveal to import it.
+There are multiple reasons for this, but a major pain point has been that Reveal does not expose the three.js examples in its export, causing applications to have to install threejs anyway.
+In addition, we also see a lot of applications ending up having multiple bundles of three.js which we are trying to alliviate.
+
+## Exploded view tool has been removed
```

Comment:
I think it's enough to have a list at the end of the document with "Removed functionality" and list the individual removed items here. They are removed because they are not used, so it's not very important to highlight them early in the doc